docs/readme.ptxt

```diff
@@ -36,85 +36,97 @@ OpenMSX is an open source Music Set for OpenTTD.
         recognizes).
         - An OpenTTD folder in your user account's home directory:
-        	Windows:
-        	  C:\My Documents (95, 98, ME)
-            C:\Documents and Settings\<username>\My Documents\OpenTTD (2000, XP)
-            C:\Users\<username>\Documents\OpenTTD (Vista, 7)
-          Mac OSX: ~/Documents/OpenTTD
-          Linux:   ~/.openttd
+            Windows:
+              C:\My Documents (95, 98, ME)
+              C:\Documents and Settings\<username>\My Documents\OpenTTD (2000, XP)
+              C:\Users\<username>\Documents\OpenTTD (Vista, 7)
+          macOS:     ~/Documents/OpenTTD
+          GNU/Linux: ~/.openttd
         - The OpenTTD installation directory.
 
-3. In the main menu of the game, click the Game Options  button. The Game
-   Options  dialog will appear.
+3. In the main menu of the game, click the Game Options button. The Game
+   Options dialog will appear.
 
-4. Select OpenMSX  from the drop-down list below Base Music set if that's not
+4. Select OpenMSX from the drop-down list below Base Music set if that's not
    selected already (bottom left of window). Close the window using the × in the
    upper left corner.
 
    Now that wasn't so hard, was it? Anyways, if you're having trouble getting
    OpenMSX to work, please file a detailed report on what you did, what error
    messages you got and where you got stuck at our bug tracker at
-   http://dev.openttdcoop.org/projects/openmsx or in the tt-forums.
+       https://github.com/OpenTTD/OpenSFX/issues
+   or at
+       https://www.tt-forums.net
 
+5. Music might not start automatically. If music does not play yet,
+   start a game and open the music box from the toolbar.
+   Here you can control the volume, and start and stop the music.
 
 
 2 Troubleshooting and bad sound output
 ======================================
 
+(Windows)
 Depending upon the hardware and the music driver used, OpenTTD may have problems
 to playback some songs without error or that it may happen that subsequent sound
 output is changed in a detrimental way. The latter is especially know to happen
 with the windows "dmusic" sound driver. You may want to try the "win32" music
-driver, if you have problems with windows' default dmusic driver. Search your
+driver, if you have problems with Windows' default dmusic driver. Search your
 openttd.cfg for "musicdriver" and change the line to "musidriver = win32".
 Consult also the OpenTTD readme for available options and its known-bugs.txt
-for a more detailed and possibly more up to date description of windows music
+for a more detailed and possibly more up to date description of Windows music
 driver issues.
 
+(GNU/Linux)
+If you cannot get music to play, it might be because your system does not
+support MIDI playback yet.
+Please refer to the manual of your Linux distribution to learn about how
+to get MIDI files to play. We recommend to install TiMidity++.
```

Comment:
I think Fluidsynth is probably preferred over Timidity (in that you can then control the volume in-game, and it's a cleaner implementation overall). Not sure this is the place for this whole section though to be honest, it should probably be in the OpenTTD readme rather than OpenMSX (since it applies to any music set).